crates/next-core/src/next_client/context.rs

```diff
@@ -238,7 +240,8 @@ pub async fn get_client_runtime_entries(
             .as_request();
 
     let mut runtime_entries = vec![RuntimeEntry::Ecmascript(
-        ProcessEnvAssetVc::new(project_root, env_for_js(env, true)).into(),
+        //TODO
```

Comment:
What's TODO?

Reply:
I guess that's solved by passing the next_config here...? Maybe @Brooooooklyn knows more.